.github/workflows/rs.yml

```
@@ -0,0 +1,25 @@
+name: Unit Testing
+
+on: [push]
+
+env:
+  CARGO_TERM_COLOR: always
+  RUST_BACKTRACE: 1
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: ./.github/actions/setup-env
+      - name: Run tests
+        run: nix develop --command just rs ci
+        working-directory: rs
+
+  msrv:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: ./.github/actions/setup-env
+      - run: nix develop --command cargo hack check --rust-version --workspace --all-targets --ignore-private
```

It would look more normal to me :D

```suggestion
      - run: nix develop --command \
      cargo hack check --rust-version --workspace --all-targets --ignore-private
```